meta-openpli/recipes-multimedia/readymedia/readymedia.bb

```diff
@@ -1,23 +1,18 @@
-DESCRIPTION = "ReadyMedia DLNA server (minidlna)"
+DESCRIPTION = "MiniDLNA a DLNA/UPnP-AV server"
 SUMMARY = "Please use the DLNAServer plugin, enigma2-plugin-extensions-dlnaserver."
-HOMEPAGE = "http://sourceforge.net/projects/minidlna"
-SECTION = "multimedia"
-MAINTAINER = "OpenPLi"
+HOMEPAGE = "http://github.com/OpenVisionE2/minidlna"
+SECTION = "network"
+MAINTAINER = "OpenVisionE2"
```

Comment:
This should be like https://github.com/OpenPLi/openpli-oe-core/blob/develop/meta-openpli/recipes-openpli/enigma2-plugins/enigma2-plugin-extensions-backupsuite.bb#L3

I mean "Open Vision Developers" ;)